src/solc_0.8/polygon/child/ChildLandBaseToken.sol

```diff
@@ -0,0 +1,85 @@
+//SPDX-License-Identifier: MIT
+/* solhint-disable func-order, code-complexity */
+pragma solidity 0.8.2;
+
+import "../../common/BaseWithStorage/ERC721BaseToken.sol";
+
+contract ChildLandBaseToken is ERC721BaseToken {
+    // Our grid is 408 x 408 lands
+    uint256 internal constant GRID_SIZE = 408;
+    // @review DO we need this?
```

Comment:
I don't think this is needed...

Reply:
you right, the contract have no need for it as it cannot mint new land out of nowhere

Reply:
Actually we use it for x and y calculation and that could be useful for external contracts

Reply:
I was more wondering about `LAYER`. But I don't see a need for it.
`GRID_SIZE` we have to keep, yes.